### DISCUSSION TITLE
Automatically associate a template type to a note and save the note in a specific location

### DISCUSSION THREAD

**Discussion by CallMeTuesday** [SilentVoid13/Templater#535] [Help]
Sorry for asking but I'm not very familiar with scripting and was wondering if someone could help me build a template plugin? I'd like the ability to create a new note using a prefixed title [[NPC-FrodoBaggins]] that triggers a new note when clicked that uses a specific template that I've created for this note type and that lives in a folder. I also need the ability to use various prefixes, for example place-, loot-, monster-

[[NPC-characterName]] yields a new note that uses my NPC.md template and lives in the NPC folder
[[NPC-placeName]] yields a new note that uses my Place.md template and lives in the Place folder

Is this something that can be done with templater?

**Comment by AB1908** [comment]
Yes, this is doable. Will need some thinking though.

**Reply by AB1908** [reply]
I did think about it and it seems tricky. Can you demonstrate with an example?

**Comment by JeromeZephyr** [comment]
Not sure about what you want to do, but you could check this, might help: [Using the Obsidian Templater Plugin - YouTube](https://www.youtube.com/watch?v=5j9fAvJCaig)

**Comment by welpdx** [comment]
Can you elaborate on the steps of your idea?
By 
> [[NPC-characterName]] yields a new note that uses my NPC.md template and lives in the NPC folder

do you mean: 
1. Click on [[NPC-characterName]]
2. On open, if active file starts with `NPC-characterName`, create a new note (where?) from the NPC.md template (which lives in some folder named NPC, i.e., foo/bar/NPC)
